feat: Create feishu-bot.yml

Think-404 opened the pull request:
增加飞书机器人actions

Think-404 committed:
Create feishu-bot.yml

xiaxuchen commented on .github/workflows/feishu-bot.yml:
```diff
@@ -0,0 +1,66 @@
+  public:
+  pull_request:
+    branches: ['main']
+    types: [opened, reopened]
```
这里 pr 应当针对于 dev 分支
其他的操作现在应该不用通知
应该还要 pr 真正合入目标分支的事件

Think-404 committed:
Create PR-Build-and-Check.yml

配置 github action，当 pr 提交的时候触发构建和检查，包括 checkstyle 检查、maven 构建检查、单元测试检查(这些直接执行 mvn 的构建就可以，只是需要配置 checkstyle 和 jacoco 的插件)

Think-404 committed:
Update feishu-bot.yml

优化了飞书机器人提示

xiaxuchen commented on .github/workflows/PR-Build-and-Check.yml:
test

Think-404 committed:
Update PR-Build-and-Check.yml

关掉checkstyle

Think-404 committed:
Update PR-Build-and-Check.yml

Think-404 committed:
Update feishu-bot.yml

将action换成我仓库的

Think-404 committed:
Update feishu-bot.yml
